mitodl/mitxonline
# Removing unused ecommerce feature flags

#### Pre-Flight checklist


#### What are the relevant tickets?

Fixes #1106 

#### What's this PR do?

Removes the `ENABLE_UPGRADE_DIALOG` and `DISABLE_DISCOUNT_UI` feature flags. 

#### How should this be manually tested?

Automated tests should pass. You should no longer be able to disable the upgrade dialog or the discount UI through your `.env` file, and the upsell cards and checkout process (including discount codes) should all work.
